### DISCUSSION TITLE
Clipping Mask

### DISCUSSION THREAD

**Discussion by felipexpert** [SVG-Edit/svgedit#975] [General]
A Clipping Mask in Photoshop is a way to control the visibility of a layer based on the shape of another layer below it. I just needed this feature to make this great tool usable in my project. Is it already possible, or do I need to create it in this existing project?

**Comment by jfhenon** [comment]
You would need to create this as we have no time to add new features. I will merge your PR if it is passing tests. If you intend to spare more time on SVGEdit, I can include you in the team.
